src/_sass/wp.scss

```diff
@@ -71,3 +71,11 @@
 }
 
 #clear { clear:both; }
+
+// Adjust typography in Bootstrap accordions
+
+#steering-faq h3 {
+  font-size: 2rem;
+  font-weight: bold;
+  margin-top: .5rem;
+}
```

Comment:
I think you may be able to get rid of this selector and instead add `.mt-0` to the accordion h3s.

```html
<h3 class="mb-0 mt-0">
```

I don't think setting the font size here is affecting the size of the text in the accordion because the `<button>` element inside of the h3 sets its own font-size to 1rem. It'd be nice if the button used `em` instead of `rem` because then your style _would_ affect it.

If you do want to boost the button size then you may need to rewrite the selector as:

```css
#steering-faq h3 > button {
  font-size: 1.3rem;
  font-weight: bold;
}
```

If you also add the `.mt-0` style to the h3 then it will end up looking like this:

![image](https://user-images.githubusercontent.com/1066253/107724251-47c73b00-6c98-11eb-8f96-00bae3b3785c.png)

Reply:
Oh, will try this, thanks!